content/docs/pulumi-cloud/deployments/_index.md

```diff
@@ -17,50 +17,64 @@ aliases:
-- [Review Stacks](/docs/pulumi-cloud/deployments/review-stacks) - dedicated cloud environments that get created automatically every time a pull request is opened.
-- Click to Deploy - trigger a deployment from the Pulumi Cloud Console with the push of a button. Very useful for day-to-day operational activities such as debugging a stuck stack, or rectifying drift via a refresh without having to pull a stack and its accompanying source code onto your machine.
-- [Scheduled Deployments](/docs/pulumi-cloud/deployments/schedules) - schedule deployment on a cron schedule or one time deployment.
-- Remote Automation API - Your favorite strongly typed Pulumi orchestration tool, Automation API, lets you offload update workloads into the cloud. Write deployment scripts that update thousands of environments simultaneously without worrying about compute.
+1. **Managed Compute**: Run Pulumi operations on secure, isolated compute instances that:
 
-Triggers can be used together, or not at all. Some users like to use the REST API to trigger deployments from GitHub Actions and disable the built-in `git push` integration. Doing so gives them the flexibility to compose deployment settings, the REST API, and click to deploy into useful platform workflows outside of CI/CD.
+   - Scale automatically with your deployment needs
+   - Handle concurrent deployments across your organization
+   - Provide detailed logs and status updates
+   - Can be replaced with your own compute resources if needed
 
-Deployment settings plus triggers are the key building blocks of the deployment platforms. With them, you can build highly customized infrastructure automation and developer platforms such as service catalogs, drift detection, temporary infrastructure workflows for cost savings, infrastructure REST APIs, and more. These scenarios and more are discussed further with demos and source code.
+1. **Flexible Triggers**: Choose how deployments are initiated:
 
-## Additional features
+   - Git push to deploy
+   - Pull request preview environments
+   - REST API calls
+   - UI-triggered deployments
+   - Scheduled operations
+   - Custom automation via the Remote Automation API
 
-- [Drift](/docs/pulumi-cloud/deployments/drift) - detect drift between the cloud and Pulumi state.
-- [Time to Live](/docs/pulumi-cloud/deployments/ttl) - configure a schedule to automatically tear down resources for a stack.
+## Getting Started
 
-## Configuring OpenID Connect (OIDC)
+The fastest way to get started with Pulumi Deployments is to:
 
-Deployments supports connectivity between Pulumi and various cloud providers via OIDC. To learn more about configuring OIDC and trust relationships in Pulumi, please refer to the [Pulumi OIDC documentation](/docs/pulumi-cloud/oidc/).
+1. Create a new project using our [New Project Wizard](link-to-wizard), which will automatically configure Deployments for your repository
```

Comment:
Is `link-to-wizard` some sort of internal magic or is this a TODO?

Reply:
It was a TODO, but I am utterly mesmerized as to why the link works. Shook, in fact.

Maybe it's because the title of the page matches the link text? Anyway, fixing to a more canonical link.